main.cc

```diff
@@ -1,4 +1,4 @@
-#include "profile.h"
+#include "profile.cc"
```

Comment:
Excuse me what

Reply:
I don't know if I'm forgetting something obvious, but when I only include the h file it doesn't compile
`/usr/bin/ld: /tmp/ccqgADwJ.o: in function `main':
main.cc:(.text+0x8a): undefined reference to `Profile::GetTime(bool)'
/usr/bin/ld: main.cc:(.text+0xbc): undefined reference to `Profile::GetTime(bool)'
collect2: error: ld returned 1 exit status`

Reply:
your code doesn't compile for me. i have to have it as `profile.h`. are you using bazel?

Reply:
I had been using g++ because I somehow missed the BUILD file, setting up tools and building now works with `profile.h`, will change